src/applications/edu-benefits/feedback-tool/components/SchoolSelectField.jsx

```diff
@@ -71,14 +70,6 @@ export class SchoolSelectField extends React.Component {
     this.debouncedSearchInstitutions.cancel();
   }
 
-  scrollToTop = () => {
-    scroller.scrollTo('schoolSearch', {
-      duration: 250,
-      delay: 0,
-      smooth: true,
-    });
-  };
-
```

Comment:
Just making sure, does this one not need to scroll to top?

Reply:
I removed it in this case just because it was not being used anywhere else in the file so it seemed to be just taking up space. However, I'll admit my familiarity with the use of this component in the application is a bit hazy, so totally open to if you see anywhere this might have a negative impact.

Reply:
Ah I see now. I just called it out because it looked different from the other files you had changed. I am probably less familiar with how this component is used than you are, so I trust your judgment here. I can see that it isn't being used or imported.